yb-voyager/cmd/templates/schema_analysis_report.txt

```diff
@@ -19,12 +19,12 @@ Migration Complexity          : {{ .MigrationComplexity }}
 Schema Summary
 --------------- 
 {{ range .SchemaSummary.DBObjects }}
-Object Type         : {{ .ObjectType }}
-  - Total Count     : {{ .TotalCount }}
-  - Valid Count     : {{ sub .TotalCount .InvalidCount }}
-  - Invalid Count   : {{ .InvalidCount }}
-  - Object Names    : {{ .ObjectNames }}{{ if .Details }}  
-  - Details         : {{ .Details }}
+Object Type                    : {{ .ObjectType }}
+  - Total Count                : {{ .TotalCount }}
+  - Valid Count                : {{ sub .TotalCount .InvalidCount }}
+  - Objects With Issues Count  : {{ .InvalidCount }}
```

Comment:
Lets name it `Objects With Issues`, Count is unnecessary.
Also lets rename `InvalidCount` variable to `ObjectsWithIssues` ?

Reply:
But before renaming `InvalidCount` we need to see if the json payloads sent to yugabyted and callhome would also change. Are we using the InvalidCount field directly there?

Reply:
I suggest renaming
TotalCount -> TotalObjects
ValidCount -> ObjectsWithoutIssues
InvalidCount -> ObjectWithIssues

Reply:
> Also lets rename InvalidCount variable to ObjectsWithIssues ?

No, we discussed and didn't do it on intentionally as it will effect json struct and will then effect yugabyted and callhome as we use it as is.
This is just UI change for HTML and TEXT reports only

Reply:
> we discussed this and didn't do it intentionally as it will effect json struct and will then effect yugabyted and callhome as we use it as is.
yeah but lets add a TODO, so that whenever we bump up the version of yugabyted payload we can consider changing this also.